Jenkinsfile

```diff
@@ -101,7 +101,36 @@ pipeline {
         branch 'master'
       }
       steps {
-        deployEcs("adminusers", "test", null, true, true)
+        deployEcs("adminusers", "test", null, false, false)
+      }
+    }
+    stage('Smoke Tests') {
+      when {
+        branch 'master'
+      }
+      steps {
+        runCardSmokeTest()
```

Comment:
card smoke test doesn't really hit adminusers
I would specify the products smoke test for now, but we probably need some sort of very light selfservice login smoke test eventually